ftl/core/exporting.ftl

```diff
@@ -38,5 +38,6 @@ exporting-processed-media-files =
         [one] Processed { $count } media file...
        *[other] Processed { $count } media files...
     }
-exporting-include-deck = Include deck
-exporting-include-notetype = Include notetype
+exporting-include-deck = Include deck name
+exporting-include-notetype = Include notetype name
+exporting-include-guid = Include unique note identifier
```

Comment:
Is note a little superfluous here, given that the context is a row/note? Not a big deal though.

Reply:
Yeah, guess you're right.